backend-api/template.yaml

```diff
@@ -225,4 +270,77 @@ Resources:
               - Effect: Allow
                 Action:
                   - kms:Sign
-                Resource: !ImportValue platform-kms-signing-key-arn
\ No newline at end of file
+                Resource: !ImportValue platform-kms-signing-key-arn
+
+# TxMA Audit service
+  TxMASQSQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 604800
+      VisibilityTimeout: 60
+      KmsMasterKeyId: !Ref TxMAKMSEncryptionKey
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt TxMASQSQueueDeadLetterQueue.Arn
+        maxReceiveCount: 5
+
+
+# TxMA account consumes this queue
+  TxMASQSQueuePolicy:
+    Type: AWS::SQS::QueuePolicy
+    Properties:
+      Queues:
+        - !Ref TxMASQSQueue
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: "Allow"
+            Action:
+              - "sqs:DeleteMessage"
+              - "sqs:GetQueueAttributes"
+              - "sqs:ChangeMessageVisibility"
+              - "sqs:ReceiveMessage"
+            Resource:
+              - !GetAtt TxMASQSQueue.Arn
+            Principal:
+              AWS:
+                - !FindInMap [TxMA, !Ref Environment, TxmaAccount]
+
+  TxMASQSQueueDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 259200 # three days
+      KmsMasterKeyId: !Sub TxMAKMSEncryptionKey
+
+  TxMAKMSEncryptionKey:
```

Comment:
I _think this should live in the KMS stack, but it means we'll need to create a new key. It doesn't in the other backend repository.

@dothomson ?

Reply:
I must have made loads of those stacks and the deletion window hasn't caused any issues. My current approach to KMS keys definitely isn't consistent. 

I don't see anything hugely wrong with having it here.

Reply:
added for TxMA

Reply:
added for all - just testing!